code/game/response_team.dm

```diff
@@ -527,6 +528,7 @@ var/ert_request_answered = 0
 	suit = /obj/item/clothing/suit/space/hardsuit/ert/engineer
 	suit_store = /obj/item/weapon/tank/emergency_oxygen/double/full
 	glasses = /obj/item/clothing/glasses/meson/night
+	belt = /obj/item/weapon/storage/belt/utility/ert/New()
```

Comment:
Not what I meant. These cannot have the New() in them. This were fine as before. What I meant was to change ert to response_team.